Cargo.toml

```diff
@@ -100,37 +100,39 @@ unused_qualifications = "warn"
 
 [features]
 default = [
-  "animation",
   "bevy_asset",
-  "bevy_state",
   "bevy_audio",
   "bevy_color",
-  "bevy_gilrs",
-  "bevy_scene",
-  "bevy_winit",
   "bevy_core_pipeline",
+  "bevy_gilrs",
+  "bevy_gizmos",
+  "bevy_gltf",
   "bevy_pbr",
   "bevy_picking",
-  "bevy_sprite_picking_backend",
-  "bevy_ui_picking_backend",
-  "bevy_gltf",
+  "bevy_remote",
   "bevy_render",
+  "bevy_scene",
   "bevy_sprite",
+  "bevy_sprite_picking_backend",
+  "bevy_state",
   "bevy_text",
   "bevy_ui",
-  "bevy_remote",
+  "bevy_ui_picking_backend",
+  "bevy_winit",
+  "android_shared_stdcxx",
+  "animation",
+  "default_font",
+  "gif",
+  "hdr",
   "multi_threaded",
   "png",
-  "hdr",
-  "vorbis",
-  "x11",
-  "bevy_gizmos",
-  "android_shared_stdcxx",
-  "tonemapping_luts",
+  "qoi",
   "smaa_luts",
-  "default_font",
-  "webgl2",
   "sysinfo_plugin",
+  "tonemapping_luts",
+  "vorbis",
+  "webgl2",
+  "x11",
```

Comment:
This was sorted since it was half-sorted, then in a random order at the end. I controversially decided to include bevy crates first, then everything else sorted alphabetically.

Only `gif` and `qoi` were added, nothing else was changed.